coconut/cmd/template_list.go

```diff
@@ -35,10 +35,31 @@ var templateListCmd = &cobra.Command{
 	Aliases: []string{"list", "ls", "l"},
 	Short: "list available workflow templates",
 	Long: `The template list command shows a list of available workflow templates.
-These workflow templates can then be loaded to create an environment.`,
+These workflow templates can then be loaded to create an environment.
```

Comment:
Examples can be added as a separate field in cobra and Long only kept as a sentence describing the use. 
```
Example: `coconut conf list
coconut conf list <component>
coconut conf list <component> -t <timestamp>`,
```

Reply:
Thanks for the tip!